<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockPage</class>
 <widget class="QWidget" name="LockPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI Light</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="syncLabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: white;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="enterBtn">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>339</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(90, 115, 227);border:none;border-radius:20px;
color: rgb(255, 255, 255);
font: 15px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic1">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>289</y>
     <width>112</width>
     <height>137</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ui/wallet_ui/login_pic1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic2">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>388</y>
     <width>85</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ui/wallet_ui/login_pic2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic3">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>126</y>
     <width>444</width>
     <height>294</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border-image: url(:/ui/wallet_ui/login_pic3.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="miniBtn">
   <property name="geometry">
    <rect>
     <x>895</x>
     <y>20</y>
     <width>11</width>
     <height>12</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,34,76);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>929</x>
     <y>20</y>
     <width>11</width>
     <height>12</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>235</y>
     <width>345</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background:transparent;border:none;border-bottom:1px solid rgb(160,160,160,153);
color: rgb(57,68,107);border-radius:0px;}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>请输入钱包密码</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic4">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>150</y>
     <width>143</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ui/wallet_ui/login_pic4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_pic1</zorder>
  <zorder>label_pic3</zorder>
  <zorder>syncLabel</zorder>
  <zorder>enterBtn</zorder>
  <zorder>label_pic2</zorder>
  <zorder>miniBtn</zorder>
  <zorder>tipLabel</zorder>
  <zorder>closeBtn</zorder>
  <zorder>pwdLineEdit</zorder>
  <zorder>label_pic4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
